src/settings/Membership/Membership.js

```diff
@@ -1,71 +1,91 @@
-import { noop } from 'lodash';
-import { FormattedMessage, useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { flow } from 'lodash';
 
+import { stripesConnect } from '@folio/stripes/core';
 import { ControlledVocab } from '@folio/stripes/smart-components';
-import { getControlledVocabTranslations } from '@folio/stripes-acq-components';
+import {
+  baseManifest,
+  getControlledVocabTranslations,
+  LoadingPane,
+} from '@folio/stripes-acq-components';
 
-import { tenants } from '../../../test/jest/fixtures/tenants';
+import { withConsortium } from '../../withConsortium';
```

Comment:
Looks like you can use the `useCurrentConsortium` hook instead of the `withConsortium` HOC.

Reply:
Thank you for reviewing @Dmytro-Melnyshyn ! Actually, HOC is used to provide `consortium` data as a prop to make it possible to use it in [manifest description](https://github.com/folio-org/ui-consortia-settings/pull/9/files#diff-92fb3797c99d3c28b0cf70d52d922b54f14c040d2a6b0cf8900f96f14b287901R65)